src/application/state.py

```diff
@@ -24,6 +19,42 @@ def __init__(self):
             # Open the welcome page
             self.current_path = NOTES_DIR + "/" + WELCOME_PAGE
 
+    def _load_settings(self) -> Dict[str, any]:
```

Comment:
this should be `typing.Any` (imported), not the `any` (standard lib) function. or just do `str` because it seems like it's just a string